### ISSUE TITLE
Unyson hasn’t been tested with the latest 3 major releases.

### ISSUE THREAD

**Issue by rafiul** [ThemeFuse/Unyson#3907]
Hi, 
I saw below alert on WP repository
![Screenshot_3](https://user-images.githubusercontent.com/6370697/73352915-5ffe2f00-42bc-11ea-8e8b-e3ae4543927a.png)

_**_This plugin hasn’t been tested with the latest 3 major releases of WordPress. It may no longer be maintained or supported and may have compatibility issues when used with more recent versions of WordPress._**_

Why didn't you change the readme? We love **Unyson**  be update pls.

**Comment by petimi** [comment]
yes, it's a bit concerning that the dev team has 350+ issues open here and seem to be abandoning this good plugin. I guess their focus is on Brizy now?
The plugin is showing PHP warning messages on PHP 7.3 and 7.4
It has multiple issues with Gutenberg editor by looking at tickets after the latest WP updates

Hopefully they will put some effort in at least compatibility updates every now and then.

**Comment by gorgeconnect** [comment]
I just discovered this with a Theme I purchased from Envato -- not to pleased, and I'm already seeing issues with 3rd party plugins such as AX Social Stream not working properly as they had before the theme switch.  Going to have to ask the theme developer if they plan on switching frameworks and if not I'm asking for a full refund.
